File: Tarea 3/Generadores/Generador.py
from random import random, randint

with open("Muestras/muestra_python_float.txt", "w", encoding="utf-8") as f:
    for _ in range(int(1e6)):
        f.write(str(random()) + "\n")

with open("Muestras/muestra_python_int.txt", "w", encoding="utf-8") as f:
    for _ in range(int(1e6)):
        f.write(str(randint(1, 6)) + "\n")

File: Tarea 3/Generadores/Generador.java
import java.io.FileWriter;
import java.io.IOException;

public class Generador {
    public static void main(String [] args){
        try(FileWriter fw = new FileWriter("Muestras/muestra_java_float.txt")){
            for(int i=0; i<1e6; i++) {
                fw.write(String.valueOf(Math.random())+"\n");
            }
        } catch (IOException e) {
            System.out.println("Error al escribir el archivo");
            e.printStackTrace();
        }
    }
}